Dockerfile

```diff
@@ -4,8 +4,7 @@ ENV TERM=xterm \
     APPLIANCE=true \
     RAILS_USE_MEMORY_STORE=true
 
-RUN dnf -y remove *subscription-manager* && \
```

Comment:
Why is this being removed?  Are you planning to use subscription-manager?

Reply:
This seems to be needed to pull from a private internal yum repository.  I think we can add this behind a conditional for now.